datasource/src/test/scala/cromwell/pipeline/database/TestMongoEngine.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.pipeline.database
+
+import cromwell.pipeline.utils.MongoConfig
+import org.mongodb.scala.{ MongoClientSettings, ServerAddress }
+
+import scala.collection.JavaConverters._
+
+class TestMongoEngine(mongoConfig: MongoConfig) extends MongoEngine(mongoConfig) {
+  override lazy val mongoSettings: MongoClientSettings = MongoClientSettings
+    .builder()
+    .applyToSslSettings(p => p.enabled(false))
+    .applyToClusterSettings(
+      p => p.hosts(List(ServerAddress("localhost", mongoConfig.port)).asJava)
```

Comment:
What is "p"? Rename it to "builder" or something else

Reply:
Fixed